tests/test_getInstcal.py

```diff
@@ -69,7 +69,7 @@ def testInstcal(self):
         self.assertEqual(visitInfo.getDate(), visit229388_info['dateAvg'])
         self.assertEqual(visitInfo.getExposureTime(), visit229388_info['exposureTime'])
         self.assertEqual(visitInfo.getDarkTime(), visit229388_info['darkTime'])
-        self.assertEqual(visitInfo.getExposureId(), int("%07d%02d" % (229388, 1)))
+        self.assertEqual(exp.info.id, int("%07d%02d" % (229388, 1)))
```

Comment:
Is there any meaningful difference between this and `exp.getInfo().getId()`, which seems to be the style of all the surrounding code? Matching the style might be better to avoid the reader even asking that question.

Reply:
The properties are recommended in new code (`.id` was specifically identified as the idiomatic accessor on DM-29955), and I felt that the inconsistency with the older C++-like code is worth it. Basically, I _want_ the reader to ask that question instead of just cargo-culting the latter. 😉

Reply:
Works for me.